projects/openweave/build.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash -eu
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+function copy_lib
+    {
+    local fuzzer_path=$1
+    local lib=$2
+    cp $(ldd ${fuzzer_path} | grep "${lib}" | awk '{ print $3 }') ${OUT}/lib
+    }
+
+mkdir -p $OUT/lib
+
+if [ "$SANITIZER" = "coverage" ]
+then
+    # so that we do not get openssl
+    export CXXFLAGS="$CXXFLAGS -fsanitize=fuzzer-no-link,address"
+    export CFLAGS="$CFLAGS -fsanitize=fuzzer-no-link,address"
+fi
+
+# build project
+./bootstrap
+# java fails with Source option 6 is no longer supported. Use 7 or later.
+./configure --disable-java --enable-fuzzing --disable-shared
+make -j$(nproc)
+find src/test-apps/fuzz/ -type f -executable -name "Fuzz*" | while read i; do
+    patchelf --set-rpath '$ORIGIN/lib' ${i}
+    copy_lib ${i} libglib
+    copy_lib ${i} libdbus
+    cp ${i} $OUT/
+done
+
+# build corpus
+ls $SRC/openweave-core/src/test-apps/fuzz/corpus/ | while read f; do
+    zip -j $OUT/${f}_seed_corpus.zip $SRC/openweave-core/src/test-apps/fuzz/corpus/${f}/*;
+done
+
+cd $OUT/
+ls *_seed_corpus.zip | grep PASE | while read c; do
```

Comment:
why just PASE? I think I see others e.g FuzzCertificateConversion

Reply:
This is just so that seed corpus zip file name matches with the fuzz target name

FuzzCertificateConversion is named ok cf 
https://github.com/openweave/openweave-core/tree/master/src/test-apps/fuzz/corpus/FuzzCertificateConversion
and 
https://github.com/openweave/openweave-core/blob/master/src/test-apps/fuzz/FuzzCertificateConversion.cpp

But PASE targets miss the `Fuzz` prefix for the corpus directory

Reply:
Ahhhh I see. Thanks, I missed that